Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Pumps/ChilledWater/Validation/Controller.mo

```diff
@@ -2,144 +2,146 @@ within Buildings.Controls.OBC.ASHRAE.PrimarySystem.ChillerPlant.Pumps.ChilledWat
 model Controller "Validate chiller water pump control sequence"
 
   Buildings.Controls.OBC.ASHRAE.PrimarySystem.ChillerPlant.Pumps.ChilledWater.Controller
-    heaNoLoc(nPum=3, nPum_nominal=3)
+    heaNoLoc(
+    nPum=3,
```

Comment:
What is the latest consensus on using `final` keyword in validation tests? I thought for these, as they validate our implementation we are still supposed to use them.